lib/ansible/modules/storage/netapp/sf_snapshot_schedule_manager.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/python
+# (c) 2017, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+
+module: sf_snapshot_schedule_manager
+
+short_description: Manage SolidFire snapshot schedules
+extends_documentation_fragment:
+    - netapp.solidfire
+version_added: '2.3'
+author: Sumit Kumar (sumit4@netapp.com)
+description:
+- Create, destroy, or update accounts on SolidFire
+
+options:
+
+    state:
+        required: true
+        description:
+        - Whether the specified schedule should exist or not.
+        choices: ['present', 'absent']
+
+    pause:
```

Comment:
not blocker: `paused` is more declarative, `pause` is imperative, we prefer the former